src/EventListener/DataContainer/TagListener.php

```diff
@@ -274,15 +274,15 @@ public function onAliasSaveCallback(string $value, DataContainer $dc): string
             $value = StringUtil::generateAlias($dc->activeRecord->name);
         }
 
-        $exists = $this->db->fetchAll("SELECT id FROM {$dc->table} WHERE alias=? AND source=?", [$value, $dc->activeRecord->source]);
+        $exists = $this->db->fetchOne("SELECT COUNT(*) FROM {$dc->table} WHERE alias=? AND source=?", [$value, $dc->activeRecord->source]);
```

Comment:
```suggestion
        $exists = (bool) $this->db->fetchOne("SELECT COUNT(*) FROM {$dc->table} WHERE alias=? AND source=?", [$value, $dc->activeRecord->source]);
```

imho :)

Reply:
We cannot convert it to boolean because the script checks if ($exists > 1) on line 280. Maybe $exists is an unfortunate naming, but I didnt want to change it.

Reply:
I will change that after merging :+1: